uber/config.py

```diff
@@ -263,6 +322,12 @@ def _is_intstr(s):
 for _badge_type, _range in _config['badge_ranges'].items():
     c.BADGE_RANGES[getattr(c, _badge_type.upper())] = _range
 
+_make_enum('badge_display', OrderedDict([(name, section['name']) for name, section in _config['badge_display'].items()]))
```

Comment:
I pretty much just copied the code for age groups to make this. We could use better support for defining dicts within dicts inside the configuration.